indico/modules/events/registration/client/js/components/ParticipantListAccordion.tsx

```diff
@@ -0,0 +1,238 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React, {HTMLAttributes, ReactNode, useState} from 'react';
+import {
+  AccordionTitle,
+  AccordionContent,
+  Accordion,
+  Icon,
+  TableRow,
+  TableHeaderCell,
+  TableHeader,
+  TableCell,
+  TableBody,
+  Table,
+  TableFooter,
+  Popup,
+  Message,
+} from 'semantic-ui-react';
+
+import {Param, Plural, PluralTranslate, Singular, Translate} from 'indico/react/i18n';
+
+import './ParticipantListAccordion.module.scss';
+
+interface TableObj {
+  headers: string[];
+  rows: [
+    {
+      checked_in: boolean;
+      columns: [{text: string; is_picture: boolean; sort_key: string | null}];
+    }
+  ];
+  num_participants: number;
+  show_checkin: boolean;
+  title?: string;
```

Comment:
Yes, when we deal with a merged table there is no title

Reply:
We could either send a default title from the back-end for merged lists, or keep it like this. Let's discuss when you're back, for me both options are okay but as it is a visual thing it might make more sense to keep like this